examples/model_selection/plot_det.py

```diff
@@ -1,7 +1,7 @@
 """
-=======================================
+====================================
```

Comment:
should  this be in `metric` instead of `model_selection` ?

Reply:
The `plot_roc` is also in this section. I am not sure what is best. I think that the idea is to compare model and select the best one in some sense. So it is not so wrong to put it in this section.